package.json

```diff
@@ -46,7 +46,8 @@
     "detect-indent": "^4.0.0",
     "esnext": "^3.1.7",
     "lines-and-columns": "^1.1.5",
-    "magic-string": "^0.22.1"
+    "magic-string": "^0.22.1",
+    "tslib": "^1.7.1"
```

Comment:
Do you think we should just do what we did for coffee-lex and decaffeinate-parser and just not use rollup?